index.js

```diff
@@ -68,7 +68,46 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector('#header')
+
+header.addEventListener('click', () => {
+  toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const likeBtn = document.querySelector('.like-button')
+const pLike = document.querySelector('.likes')
+
+likeBtn.addEventListener('click', () => {
+  const likes = parseInt(pLike.textContent)
+  pLike.textContent = likes + 1 + " Likes"
+})
+
 
 /***** Deliverable 3 *****/
+
+
+//link https://youtu.be/6rmUwoM-KQs?t=131
+
+const form = document.querySelector('#new-goal-form')
+const goalLinkForm = form.querySelector('#goal-link')
+const goalDescription = form.querySelector('#goal-description')
+
+form.addEventListener('submit', event => {
+  event.preventDefault()
+
+  const link = event.target.link.value
+  const description = event.target.description.value
+  
+  const goalId = document.querySelector('#goals').lastChild.dataset.id
+  const newId = parseInt(goalId) + 1
+
+  const newGoal = {
+    id: newId,
+    link: link,
+    description: description
```

Comment:
If you want to save on variables declarations, you could also create your object like so:
```javascript
  const newGoal = {
    id: parseInt(goalId) + 1,
    link: event.target.link.value,
    description: event.target.description.value
  }
```

Another way of getting id to update is to get it from the latest goal in the goals array instead.